src/Wrapper/Array.cs

```diff
@@ -163,6 +164,881 @@ public Type ElemType
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		public static Array operator ^(Array lhs, Array rhs) { IntPtr ptr; Internal.VERIFY(AFArith.af_bitxor(out ptr, lhs._ptr, rhs._ptr, false)); return new Array(ptr); }
 #endif
+
+#if _
```

Comment:
I have only one question in this file's change. I can understand what the logic under the `#if`'s arms are doing, but what does this `#if _` exactly check for ?

Reply:
These directives are actually intended to be used by the F# script in AutoGenTool. Everything inside `#if _` and `#else` is the "template" for code generation, and the generated code will appear between `#else` and `#endif`